robottelo/host_helpers/contenthost_mixins.py

```diff
@@ -40,7 +40,34 @@ def OSCAP(self):
             'cbrhel': constants.OSCAP_PROFILE[f'cbrhel{self._v_major}'],
         }
 
-    # pull from settings.server.repos???
+    def dogfood_repofile(self, product=None, release=None, snap=''):
```

Comment:
For 6.11.z, we already have the previous download_repos helper to download repos, so this func will be redundant for this branch OR else consider replacing download_repos with this and change usage